ci/test-conformance-eks.sh

```diff
@@ -134,6 +134,9 @@ ${AWS_SECRET_KEY}
 ${REGION}
 JSON
 EOF
+    echo "=== Upgrading eksctl version if needed ==="
```

Comment:
```suggestion
    echo "=== Installing latest version of eksctl ==="
```

It seems that we always install it, regardless of whether or not we need to

Reply:
done